src/utils.jl

```diff
@@ -92,15 +92,22 @@ end
 compute_offset(ŷ::AbstractVector{<:Distributions.UnivariateDistribution}) = expected_value(ŷ)
 compute_offset(ŷ::AbstractVector{<:Real}) = expected_value(ŷ)
 
-function compute_covariate(jointT, W, G, indicator_fns; threshold=0.005)
-    # Compute the indicator values
-    indic_vals = TMLE.indicator_values(indicator_fns, jointT)
-    # Compute density and truncate
+function balancing_weights(G::Machine, W, jointT; threshold=0.005)
     ŷ = MLJBase.predict(G, W)
     d = pdf.(ŷ, jointT)
     plateau!(d, threshold)
-    indic_vals ./= d
-    return indic_vals
+    return 1. ./ d
+end
+
+function covariate_and_weights(jointT, W, G, indicator_fns; threshold=0.005, weighted_fluctuation=false)
+    # Compute the indicator values
+    indic_vals = TMLE.indicator_values(indicator_fns, jointT)
+    w = balancing_weights(G, W, jointT, threshold=threshold)
+    if weighted_fluctuation
```

Comment:
Would be good to link the webpage: https://lendle.github.io/TargetedLearning.jl/user-guide/ctmle/#fluctuating-barq_n 
(or better: write this in the documentation and link it)

Reply:
Could you please ask Mark for the source of this by the way, I can't find anything beside the package which does not provide any justification for it?

Reply:
Good point, the closest I could find for now is section 3.3 in https://www.degruyter.com/document/doi/10.1515/1557-4679.1334/html (albeit in the context of of temporal data, so it's still not proved here but at least it has been used). I will keep looking and let you know.